src/platform/linux/kmsgrab.cpp

```diff
@@ -1684,8 +1684,11 @@ namespace platf {
 
         if (!fb->handles[0]) {
           BOOST_LOG(error) << "Couldn't get handle for DRM Framebuffer ["sv << plane->fb_id << "]: Probably not permitted"sv;
-          BOOST_LOG((window_system != window_system_e::X11 || config::video.capture == "kms") ? fatal : error)
+          auto loglevel = (window_system != window_system_e::X11 || config::video.capture == "kms") ? fatal : error;
+          BOOST_LOG(loglevel)
             << "You must run [sudo setcap cap_sys_admin+p $(readlink -f $(which sunshine))] for KMS display capture to work!"sv;
+          BOOST_LOG(loglevel)
```

Comment:
I don't think this should be a separate log entry. It could just be added to the message above this.

Reply:
I had that in a first iteration, but the layout in the logfile looked kinda bad: In order to avoid an overly long line I ended the current message with a `\n` and then spit out the additional text in a new line. But then that line was not preceded by the timestamp, and consequently it was misaligned.

That's why I opted to emit two seperate lines. (I could change it back though.)

Reply:
Ok, I've changed it. I also added the URL. In case of a fatal failure it looks like this now:
![image](https://github.com/LizardByte/Sunshine/assets/11065156/af932d30-8d48-4c54-a45a-8b7ac53b0b6e)